DEVELOPMENT.md

```diff
@@ -11,6 +11,27 @@ See our [Contributing Guide](CONTRIBUTING.md) for tips on how to submit a pull r
 # Initial Development Environment Setup
 Athens relies on having a few tools installed locally. Run `make setup-dev-env` to install them.
 
+### Go version
+Athens is developed on Go1.11+.
```

Comment:
We still have the ability to develop Athens with Go 1.10.x right now, but the proxy requires a Go 1.11+ binary to function properly.

Maybe this can say something like "Athens requires Go 1.11+ to be installed, but you can develop using 1.10+"

Not sure if that wording is confusing...

Reply:
Yeah! I would stick to go1.11+ as you can't really test without it. I have spent so many hours not realizing that you can develop on version but can't really test on it :-(. Let me know if I need to update. Can olympus run on go1.10, if so, the go1.11 docs can be moved to Proxy.md

Reply:
THere's basically two different things.
Version you use to build the project
Version you use to work with modules.

First one i would keep as is, not specify something which is not true. I do development on 1.10.
Second one needs to be 1.11 because it has features we're working with (modules), this you specify using GO_BINARY_PATH.

I would say just like it is so it's not confusing. We do not require folks to use unstable go release for development for now.

Reply:
Sounds good to me! Thanks @manugupt1 and @michalpristas for explaining